sf_examples/nethack/README.md

```diff
@@ -0,0 +1,53 @@
+## Installation
+
+To install NetHack, you need nle and its dependencies.
+
+```bash
+# nle dependencies
+apt-get install build-essential python3-dev python3-pip python3-numpy autoconf libtool pkg-config libbz2-dev
+conda install cmake flex bison lit
+
```

Comment:
My preference would be to avoid duplicating the README as you'd need to update & maintain both files.
Can we have a very brief readme here that just points at documentation?

Reply:
That's true, I've added the documentation later and forgot to remove it. Done.